gh_pages/_source/session5/Simple-PCL-Interface-for-Python.rst

```diff
@@ -300,6 +300,12 @@ Viewing Results
 
             rosrun filter_call filter_call.py _pcdfilename:="/home/ros-industrial/catkin_ws/table.pcd"
 
+#. Source a new terminal and run the tf2_ros package to publish a static coordinate transform from the child frame to the world frame
+
+   .. code-block:: bash
+   
+            rosrun tf2_ros static_transform_publisher 0 0 0 0 0 0 0 world_frame kinect_link
```

Comment:
This line creates an invalid quaternion; please change this line to a valid unit quaternion [0, 0, 0, 1]:
```
rosrun tf2_ros static_transform_publisher 0 0 0 0 0 0 1 world_frame kinect_link
```
or use the roll-pitch-yaw implementation (i.e. one fewer zero)
```
rosrun tf2_ros static_transform_publisher 0 0 0 0 0 0 world_frame kinect_link
```